resources/tasks/docker/remove_docker_container.yml

```diff
@@ -19,5 +19,5 @@
     force_kill: "{{ lookup('vars', _var_prefix + '_docker_force_kill', default=omit) }}"
     name: "{{ lookup('vars', _var_prefix + '_docker_container') }}"
     state: absent
-    stop_timeout: "{{ lookup('vars', _var_prefix + '_docker_stop_timeout', default='10') }}"
+    stop_timeout: "{{ lookup('vars', _var_prefix + '_docker_stop_timeout', default='30') }}"
```

Comment:
Why? Roles can override this if they need more time.

Reply:
This could be set to a variable that contains the default value instead to allow inventory overrides but it is not a good idea to bump this by default